ebmbot/slack.py

```diff
@@ -1,11 +1,22 @@
 from time import sleep
 
+from slack_sdk import WebClient
+
 from ebmbot import settings
 from workspace.utils.blocks import get_basic_header_and_text_blocks, truncate_text
 
 from .logger import logger
 
 
+def slack_web_client(token_type="bot"):
+    token = (
+        settings.SLACK_BOT_TOKEN
+        if token_type == "bot"
+        else settings.SLACK_BOT_USER_TOKEN
+    )
```

Comment:
Micro-nit, but I'd be tempted to write this in some way which errors if it isn't given an explicitly expected value. Could do it with a dict e.g.
```suggestion
   tokens = {
       "bot": settings.SLACK_BOT_TOKEN,
       "user": settings.SLACK_BOT_USER_TOKENm
   }
   token = tokens[token_type]
```
An `if ... elif ... else: assert False` chain would also do the trick, but a bit more cumbersome.